.github/workflows/cr.yml

```diff
@@ -0,0 +1,35 @@
+name: Continuous Releases
+
+on:
+  push:
+    branches:
+      - main
+      - docusaurus-v**
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    name: Continuous Releases
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Installation
+        run: yarn
+
+      - name: Build packages
+        run: yarn build:packages
+
+      - name: Fresh templates
+        run: |
+          yarn create-docusaurus fresh-examples/docusaurus-classic classic --javascript -p npm -s 
```

Comment:
We do not need installs, since templates would just be uploaded! 

I also use npm here so we avoid that error, at least until I fix the issue with yarn in pkg.pr.new!

Reply:
Hmmm, wondering if generating a package-lock.json wouldn't be useful?

It looks like a tradeoff: resolve package versions ahead of time in CI vs letting the stackblitz user handle it. Maybe the DX will be better/faster for the end user if versions are already pre-resolved?